modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -190,30 +196,23 @@ Specify the following information:
 | bpm
 |===
 
-[NOTE]
-====
-
-Note: If you are using MySQL, add `?allowMultiQueries=true` to the URL. For example,
-    `db.url=jdbc:mysql://localhost:3306/bonita_migration?allowMultiQueries=true`. +
-    Also, if you are migrating to Bonita 7.9+, you must upgrade your database server to MySQL 8.0, see <<mysql8,Migrating to Bonita 7.9+ using MySQL>> specific procedure below.
-====
 
-. If you use a custom Look & Feel, xref:managing-look-feel.adoc[export] it, and then xref:managing-look-feel.adoc[restore the default Look & Feel].
-. Generally speaking, pausing (or not) the BPM services before migrating should not impact migration. There are however a few special cases:
- ** If you are migrating from a version *< 7.3.0*. If you are using a BDM, your BDM will have to be redeployed after migration (see <<bdm_redeploy,above>>).
-In this case, it is recommended you *stop your BPM services before migrating*, so as to be able to redeploy your BDM immediately after migration (and not having to deal with eventual errors in automated processes as you restart your platform after migration).
- ** Several bugs affect legacy versions of Bonita that prevent a smooth migration of a Bonita platform with *BPM services paused* in special cases:
-  *** Starting version is *before 7.8.0*, and your target version is between *7.8.0* and *7.11.5*
-  *** Starting version is *before 7.10.5* & target version is *before 2021.1*
- ** Therefore, while it is recommended to always migrate to the latest version of Bonita, if your target version is not 2021.1, or above, it is necessary to migrate with *BPM services running*.
-. Stop the application server.
-. *IMPORTANT:*
-xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in case of problems during migration.
-. Go to the directory containing the migration tool.
-. Run the migration script:
+7. Custom Portal Look & Feel: 
+ * Target version older than Bonita 2021.2: If you used a custom Look & Feel in Bonita Portal, export it, and then restore the default Look & Feel.
```

Comment:
Same feedback, if this doesn't apply to the 2021.2 version then remove it from the 2021.2 documentation branch.

Reply:
You cannot restart at 7 after stoping the enumeration, asciidoctor doesn't work this way. If you would have checked the preview you would have noticed that it went back to 1.

Reply:
I saw that and asked for help to solve this issue.

Reply:
Removed the "older than Bonita 2021.2" part